src/pages/new-task.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <h1>Create new task</h1>
+    <form @submit.prevent>
+      <div class="form-group">
+        <label>Title:</label>
+        <input v-model="card.title" class="form-control" type="text">
+      </div>
+      <div class="form-group">
+        <label>Description:</label>
+        <textarea v-model="card.descr" class="form-control"></textarea>
+      </div>
+      <button :disabled="!card.title || !card.descr" @click="addNew" class="btn btn-primary">Submit</button>
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+
+  data() {
+    return {
+      card: {
+        title: null,
+        descr: null,
+        id: null
+      }
+    }
+  },
+  methods: {
+    addNew() {
+      this.$store.dispatch('addCard', {title: this.card.title, descr: this.card.descr, id: (Date.now() + Math.random() * 10e10).toString(16).split('.').shift()});
```

Comment:
We discussed that form component doesn't have to pass id to a store. It has to happened inside the store